modules/ROOT/pages/index.adoc

```diff
@@ -23,6 +23,162 @@ Anypoint Platform for PCF uses the following services:
+.. After the app is created, run `cf app <app>` and make note of the app URL.
+. Deploy a proxy application:
+               
+.. Use the PCE instance to get properties.
++
+----
+-Dapi.name=<customerApp name>     
+-Dapi.version=<version of the app ex: 1.0>    
+-Dimplementation.host=<url of customerapp ex: springboot.apps.santafesprings.cf-app.com>
+-Dimplementation.port=80 
+-Dimplementation.path=/ 
+-Danypoint.platform.client_id=<clientId property that you have for the org ex: cdbff39bfa4741fb9bb531ba6102a578> 
+-Danypoint.platform.client_secret=<ClientId secret of the org ex: 56bB958Bc257464B9fF0A9F9E6Af9890> 
+-Danypoint.platform.platform_base_uri=< ex: https://qax.anypoint.mulesoft.com/apiplatform>
+-Danypoint.platform.coreservice_base_uri=< ex: https://qax.anypoint.mulesoft.com/accounts >
+-Danypoint.platform.on_prem=true 
+-Danypoint.platform.contracts_base_uri=< ex: https://qax.anypoint.mulesoft.com/apigateway/ccs>
+----
+
+.. After gathering all the properties, deploy the proxy:
++
+----
+. cf push <proxy> -b <mulebuildpackname> -p <proxy app that mulesoft provided> --no-start
+. cf set-env <proxy-app> mcm_anypoint_metering_plan_unlimited true
+. cf set-env <proxy-app> MULE_JAVA_OPTS "-Dapi.name=muleapp -Dapi.version=1.0
+-Dimplementation.host=muleapp.apps.santafesprings.cf-app.com -Dimplementation.port=80
+-Dimplementation.path=/ -Danypoint.platform.client_id=0f2ea15d88fa452e8ca1a5f4bb724728
+-Danypoint.platform.client_secret=BbCA70Bb056F437e94C707B67aF4284D
+-Danypoint.platform.platform_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/apiplatform
+-Danypoint.platform.coreservice_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/accounts -Danypoint.platform.on_prem=true
+-Danypoint.platform.contracts_base_uri=https://161-171-upgrade-2056596225.us-east-2.elb.amazonaws.com/apigateway/ccs
+
+. cf start <proxy>
+----
+
+.. Tail the logs by running `cf logs <proxy>`.
+
+***  The 2018-11-13T10:07:09.52-0800 [APP/PROC/WEB/0] OUT INFO  2018-11-13 18:07:09,494 [agw-policy-polling.01] com.mulesoft.module.policies.lifecyle.GateKeeper log should indicate that API <proxyappname> <version> is now unblocked.
+
+*** In the PCE instance of API manager, the proxy auto registers with status as ACTIVE. If you see `401` or `500` errors in the logs, make sure you have sufficient permissions to create an API in API manager. The proxy uses the properties provided in the previous step.
+
+.. Create a user service instance for the proxy app using the following command:
++
+----
+`cf create-user-provided-service SERVICE-INSTANCE -r ROUTE-SERVICE-URL`
+----
+... For example, `cf create-user-provided-service muleappproxy -r https://muleappproxy.apps.santafesprings.cf-app.com`.
+... For additional information, refer to https://docs.cloudfoundry.org/services/route-services.html#user-provided[User-Provided Service, role=external, window=_blank].
```

Comment:
Why are these formatted as steps?
The first item is a fragment and should be a complete sentence.